controllers/manifests/base/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sa
+  annotations:
+    # This annotation is used by OpenShift to assign the anyuid SCC
+    # which allows the container to run as any user ID
+    openshift.io/scc: anyuid
```

Comment:
I think this comment needs to at least explain _why_ the container needs any user id; like which dependency and why